src/main/java/org/spongepowered/api/util/Color.java

```diff
@@ -38,44 +36,68 @@
 import org.spongepowered.api.data.persistence.InvalidDataException;
 import org.spongepowered.api.data.type.DyeColor;
 
-import java.util.Objects;
 import java.util.Optional;
 
+import static com.google.common.base.Preconditions.checkArgument;
+
 public final class Color implements DataSerializable {
 
-    private static final int MASK = 0xFF;
+    //16 HTML web colors and CSS orange
+
+    public static final Color WHITE = ofHex(0xFFFFFF);
+
+    public static final Color SILVER = ofHex(0xC0C0C0);
+
+    public static final Color GRAY = ofHex(0x808080);
+
+    public static final Color BLACK = ofHex(0x000000);
+
+    public static final Color RED = ofHex(0xFF0000);
 
-    public static final Color BLACK = ofRgb(0x000000);
+    public static final Color MAROON = ofHex(0x800000);
 
-    public static final Color GRAY = ofRgb(0x808080);
```

Comment:
Just don't change these, its an absolutely pointless breaking change.

Reply:
This change is one of the largest reasons this PR exists. There is no reasoning behind the existing colors offered in the classes, and this attempts to standardize them to the common web colors.

It is also not a breaking change. Unused fields have been marked `@Deprecated` to be removed later, and the only field that now represents a different color is Purple, which is a shade change.

Reply:
And when they are removed that is the breaking change. What is the point of removing a few colors from what is already a completely arbitrary set of colors?

Reply:
To make it so the colors are *not* arbitrary. Though it is possible to retrieve colors from either `TextColor`s or `DyeColor`s, I think it's reasonable to have some standardized colors (especially those that aren't available above or used in multiple places through API and Common, like black) offered to developers. I've opted to use the 17 CSS named colors, which are the 16 HTML colors and orange.